### PR TITLE
[LIBS-433] Add single table utilities

### PR DESCRIPTION
**Pull request by smilin-desperado**

### CODE REVIEW

File: src/dynamodb-container.ts [hunk 1]
```diff
@@ -23,13 +23,21 @@ import {
...
 
 export class StartedDynamoDBContainer implements StartedTestContainer {
+  private tables: string[] = [];
+
   constructor(
     private readonly startedContainer: StartedTestContainer,
     private readonly initData: Array<TableInitStructure>
   ) {}
 
+  /** Test table names creatd */
```
Comment (Thris3n) [inline src/dynamodb-container.ts]: Typo

File: src/dynamodb-container.ts [hunk 2]
```diff
@@ -194,17 +275,22 @@ export interface TableInitStructure {
...
 
 export class DynamoDBContainer extends GenericContainer {
   private static readonly IMAGE_NAME = 'amazon/dynamodb-local';
   public static readonly MAPPED_PORT = 8000;
+  private shouldResetData: boolean;
 
   constructor(
     private readonly initStructure: Array<TableInitStructure> = [],
-    { reuse }: DynamoDBContainerOptions = {}
+    { reuse, reset = true }: DynamoDBContainerOptions = {}
```
Comment (Thris3n) [inline src/dynamodb-container.ts:290]: Reset should be true?

**Review by Thris3n — CHANGES_REQUESTED** [review]

File: src/dynamodb-container.ts (hunk 1 above)
Comment (smilin-desperado) [reply src/dynamodb-container.ts]: Touché 😆

File: src/dynamodb-container.ts (hunk 2 above)
Comment (smilin-desperado) [reply src/dynamodb-container.ts:290]: Yes! The current behaviour is for reset to always happen, adding a default of true means it shouldn't break existing implementations but allows a new implementation to opt in to the new behaviour

**Review by Thris3n — APPROVED** [review]

**Pull request merged**